### DISCUSSION TITLE
400 API error when patching single row [Feb 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5264] [Discord Threads]
> 🙋‍♂️ **pain peko**

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### Are you referring to this? https://all-apis.nocodb.com/#tag/DB-table-row/operation/db-table-row-update   
#### The path would go something like this:   
#### ```   
#### {   
####   orgs: 'string',    
####   projectName: 'string',   
####   tableName: 'string',   
####   rowId: 'string'   
#### }   
#### ```   
#### Given its a 400 error it means your request needs modification.  
#  
> 🙋‍♂️ **pain peko**  
#### i mean this https://all-apis.nocodb.com/#tag/DB-table-row/operation/db-table-row-update  
#  
> 🙋‍♂️ **pain peko**  
####   
![image0](https://cdn.discordapp.com/attachments/1079193974264307762/1079246824273104967/image.png)  
#  
> 🙋‍♂️ **pain peko**  
#### this is the image when i try to update my json field  
#  
> 🙋‍♂️ **pain peko**  
#### is my payload wrong somewhere?  
#  
> 🙋‍♂️ **kyleecodes**  
#### I've only used a one-dimensional json body for the api calls, so instead of "cache" -> "test" its just "test". What type of data are you trying to update with? Tagging <@&682859083799199814>  
#  
> 🙋‍♂️ **pain peko**  
#### my table 'Weather_cache' in postgres have a column name 'cache' type json, i want to update that column with rest api. I'm how sure how to use REST API to do that since there nothing about it in the swagger api and i didn't find anything in the doc  
#  
> 🙋‍♂️ **wkw**  
#### probably u need stringify the payload.   
####    
#### ```   
#### {   
####   "cache": "{\n  \"test\": \"sample message\"\n}"   
#### }   
#### ```  
#  
> 🙋‍♂️ **pain peko**  
#### that work, thank you so much  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1079193974264307762)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
